modules/core/test/test_mat.cpp

```diff
@@ -2259,6 +2259,23 @@ TEST(Core_Eigen, eigen2cv_check_Mat_type)
     EXPECT_ANY_THROW(eigen2cv(eigen_A, d_mat));
     //EXPECT_EQ(CV_64FC1, d_mat.type());
 }
+
+TEST(Core_Eigen, cv2eigen_check_RowMajor)
+{
+    Mat A(2, 2, CV_32FC1, Scalar::all(0));
```

Comment:
> 2, 2

It makes sense to use different values for width/height (e.g. 3,2) to verify argument order issue.

Reply:
Fixed